src/services/upload_repository.js

```diff
@@ -10,28 +10,26 @@ This Source Code Form is “Incompatible With Secondary Licenses”, as defined
 
 import {ValidationError} from '../errors/errors';
 import {Role} from './roles_repository';
-import BN from 'bn.js';
-import * as Sentry from '@sentry/node';
+import {InsufficientFundsToUploadBundleError} from 'ambrosus-node-contracts';
 
 export default class UploadRepository {
-  constructor(web3, identityManager, uploadsActions, shelteringWrapper, rolesWrapper, feesWrapper, configWrapper, lowFundsWarningAmount) {
+  constructor(web3, identityManager, uploadsActions, shelteringWrapper, rolesWrapper, configWrapper, lowFundsWarningAmount, sentry) {
     this.web3 = web3;
     this.identityManager = identityManager;
     this.uploadsActions = uploadsActions;
-    this.feesWrapper = feesWrapper;
     this.shelteringWrapper = shelteringWrapper;
     this.configWrapper = configWrapper;
     this.rolesWrapper = rolesWrapper;
     this.lowFundsWarningAmount = lowFundsWarningAmount;
+    this.sentry = sentry;
   }
 
   async ensureBundleIsUploaded(bundleId, storagePeriods) {
     await this.validateNodeIsOnboardedAsHermes();
-    await this.validateEnoughFundsForUploadFee(storagePeriods);
 
     const previousUploader = await this.shelteringWrapper.getBundleUploader(bundleId);
     if (this.isEmptyAddress(previousUploader)) {
-      return this.receiptWithResult(this.uploadsActions.uploadBundle(bundleId, storagePeriods), 'Bundle has been uploaded');
+      return this.receiptWithResult(this.uploadsActions.uploadBundle(bundleId, storagePeriods, this.lowFundsWarningAmount), 'Bundle has been uploaded');
```

Comment:
`lowFundsWarningAmount` should be passed to the UploadActions' constructor, this shall not work now